browser/js/common/factories/addressFactory.js

```diff
@@ -0,0 +1,24 @@
+//sv
+app.factory("AddressFactory", function () {
+    //sv (str, str) => string
+    function getFullName(firstname, lastname) {
+        return capatilize(firstname) + " " + capatilize(lastname);
```

Comment:
it's spelled "capitalize" not "capitilize" but if we don't feel like doing a find & replace i'm certainly not worried